openlibrary/templates/lib/nav_head.html

```diff
@@ -105,6 +105,11 @@
         </ul>
       </div>
     </div>
+    <div class="search-barcode-component">
+      <a href="https://openlibrary.org/barcodescanner?returnTo=/isbn/$$$" id="barcode_scanner_link" class="search-barcode-component-link">
+        <img src="../../../static/images/icons/barcode_scanner.svg" alt="Search by barcode" width="100px" height="80%"/>
```

Comment:
for alt text, should I google translate it in the available languages, and submit a CR with the translations? (following this readme? https://github.com/internetarchive/openlibrary/tree/master/openlibrary/i18n)

Reply:
```suggestion
        <img src="../../../static/images/icons/barcode_scanner.svg" alt="$_('Search by barcode')" width="100px" height="80%"/>
```

All you have to do is this! Translators will handle the rest of the flow 😊 The `_` function marks this text as needing translation.